content/en/opentelemetry/otlp_metrics.md

```diff
@@ -0,0 +1,284 @@
+
+- **Technical constraints using the Datadog Exporter or Agent**: Ideal for scenarios where installing additional software is impractical or restrictive, such as third-party managed services (for example, Vercel), applications on customer devices, or environments requiring streamlined, Agentless observability pipelines. The OTLP metrics endpoint enables direct OTLP metric ingestion in these scenarios.
+
+## Configuration
+
+To export OTLP metrics data to the Datadog OTLP metrics intake endpoint:
+
+1. [Ensure only delta metrics are sent](#ensure-delta-temporality).
+1. [Configure the OTLP HTTP exporter](#configure-the-exporter).
+   - Set the Datadog OTLP metrics intake endpoint.
+   - Configure the required HTTP headers.
+1. (Optional) [Set the `dd-otel-metric-config` HTTP header](#optional-configure-the-metric-translator) to configure the metric translator behavior.
+
+### Configure the exporter
+
+To send OTLP data to the Datadog OTLP metrics intake endpoint, you need to use the OTLP HTTP Protobuf exporter. The process differs depending on whether you are using automatic or manual instrumentation for OpenTelemetry.
+
+#### Ensure only delta metrics are sent
+
+The Datadog OTLP metrics intake endpoint accepts only **delta** metrics. If you attempt to send **cumulative** metrics (the default in most SDKs), you will receive an error. Make sure to configure your OpenTelemetry SDK or Collector to produce delta metrics.
+
+- For [supported languages][7], set the `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` environment variable to `delta`.
+- Otherwise, set the temporality preference in your code. For more information, read [Configure delta temporality in OpenTelemetry][5].
+
+#### Automatic instrumentation
+
+If you are using [OpenTelemetry automatic instrumentation][3], set the following environment variables:
+
+```shell
+export OTEL_EXPORTER_OTLP_METRICS_PROTOCOL="http/protobuf"
+export OTEL_EXPORTER_OTLP_METRICS_ENDPOINT="{{< region-param key="otlp_metrics_endpoint" >}}"
+export OTEL_EXPORTER_OTLP_METRICS_HEADERS="dd-api-key=${DD_API_KEY}"
+export OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE="delta"
+```
+
+#### Manual instrumentation
+
+If you are using manual instrumentation with OpenTelemetry SDKs, configure the OTLP HTTP Protobuf exporter programmatically.
+
+<div class="alert alert-info">Based on your <a href="/getting_started/site/">Datadog site</a>, which is {{< region-param key=dd_datacenter code="true" >}}: Replace <code>${YOUR_ENDPOINT}</code> with {{< region-param key="otlp_metrics_endpoint" code="true" >}}.</div>
+
+{{< tabs >}}
+{{% tab "JavaScript" %}}
+
+The JavaScript exporter is [`@opentelemetry/exporter-metrics-otlp-proto`][100]. To configure the exporter, use the following code snippet:
+
+```javascript
+const { OTLPmetricExporter } = require('@opentelemetry/exporter-metrics-otlp-proto'); 
```

Comment:
```suggestion
const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-proto'); 
```
Per [the package documentation](https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-proto) (I only noticed because the casing as written is not idiomatic).